src/js/utils/notifier.js

```diff
@@ -24,16 +24,22 @@ const filterReport = report => {
   return scripts.some(script => report.stacktrace.some(includes(script)))
 }
 
-const bugsnagClient = bugsnag({
-  apiKey: process.env.BUGSNAG_API_KEY,
-  appVersion: getAppVersion(),
-  collectUserIp: false,
-  beforeSend: filterReport,
-  releaseStage: process.env.NODE_ENV,
-  notifyReleaseStages: ["production"]
-})
-
-bugsnagClient.use(bugsnagReact, React)
-
-export default bugsnagClient
```

Comment:
@manubo this default `export` now is gone, might this be an issue?

Reply:
@pic No, it's not an issue. The default export was not used anywhere, I check before removing it, but it's great you asked 👍